sfs/util.py

```diff
@@ -491,3 +496,26 @@ def _count_walls_1d(a):
         wall_count = wall_count[N_mask, :]
 
     return xs, wall_count
+
+
+def spherical_hn2(n, z):
+    r"""Spherical Hankel function of 2nd kind.
+
+    Defined as http://dlmf.nist.gov/10.47.E6,
+
+    .. math:: \hankel{2}{n}{z} = \sqrt{\frac{\pi}{2z}} \Hankel{2}{n + 1/2}{z},
```

Comment:
I think `\frac{1}{2}` would be clearer, because otherwise one might be tempted to think at first glance it's `(n+1)/2`.